nokemobilelibrary/src/main/java/com/noke/nokemobilelibrary/NokeDeviceManagerService.java

```diff
@@ -112,7 +114,18 @@
      * A boolean that allows the device manager to discover devices that are not in the array
      */
     private boolean mAllowAllDevices;
-
+    /**
+     *property used to detect if a connection fails when a device that is not available tries to create a connection
+     */
+    public Handler connectionTimer;
+    /**
+     *number of seconds the connection process will wait until return an error connection
+     */
+    public long numberOfSecondsToDetectTheConnectionError = 2;
```

Comment:
We are using 2 seconds here, but if you consider this should be different just let me know

Reply:
Android devices often take longer to connect, so I recommend increasing this timeout. I'd go with a minimum of 6 secs. Maybe more if you start to see the timeout error popping up frequently during testing

Reply:
I'll create another PR with these changes, thanks